app/uk/gov/hmrc/serviceconfigs/Module.scala

```diff
@@ -45,7 +45,7 @@ class Module extends play.api.inject.Module {
   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] =
     Seq(
       bind[ConfigScheduler               ].toSelf.eagerly()
-    , bind[BobbyWarningsNotifierScheduler].toSelf.eagerly()
+    , bind[SlackNotificationScheduler    ].toSelf.eagerly()
```

Comment:
Maybe call `WarningsNotifierScheduler` (maybe even `DeprecationWarningsNotfierScheduler`)? Since the objective is to raise warnings and slack-notifications is just the medium?